cmd/client.go

```diff
@@ -173,6 +175,54 @@ func (cli *Client) BackupDatabase(c *clipkg.Context) error {
 	return cli.errorOut(saveBodyAsFile(resp, c.Args().First()))
 }
 
+// ImportKey imports a key to be used with the chainlink node
+func (cli *Client) ImportKey(c *clipkg.Context) error {
+	cfg := cli.Config
+	if !c.Args().Present() {
+		return cli.errorOut(errors.New("Must pass in filepath to key"))
+	}
+
+	src := c.Args().First()
+	dst := cfg.KeysDir()
+	if !isdirEmpty(dst) {
+		return cli.errorOut(fmt.Errorf("Account already present in %s", dst))
+	}
+	dst += src[strings.LastIndex(src, "/"):]
```

Comment:
Need to test this more later today. Im sure it doesnt work for something like chainlink import key.json